packages/zapp/console/src/components/Executions/TaskExecutionsList/utils.ts

```diff
@@ -61,3 +61,9 @@ export const getGroupedLogs = (resources: Event.IExternalResourceInfo[]): LogsBy
 
   return logsByPhase;
 };
+
+export function getTaskLogName(taskName: string, taskLogName: string): string {
```

Comment:
getTaskLogName should be created from

- `taskName` - maybe it even make sense to pick only part after last dot, as we do in the graph and a lot of other places (but it's your call)
- 'externalResources[n].index' or 0 if not present
- 'externalResources[n].retryAttempt' or 0 if not included

Sample of one of externalResources items:
<img width="488" alt="Screen Shot 2022-05-16 at 6 58 52 PM" src="https://user-images.githubusercontent.com/55718143/168712982-06fb4100-5eb1-441f-89f5-7116e3622f6a.png">